frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx

```diff
@@ -63,6 +63,8 @@ const ImageSearch: React.FC = () => {
           setFieldValue('isi.ports', ports);
           setFieldValue('image.ports', ports);
           values.name === '' && setFieldValue('name', getSuggestedName(name));
+          values.application.name === '' &&
```

Comment:
In one spot it checks `!values.application.name` and here it does `values.application.name === ''`. 
The checks should be consistent.

Reply:
will update it, I did based on above line `values.name === ''`